_includes/side_menu.html

```diff
@@ -1,6 +1,6 @@
 <h2>{{ include.title }}</h2>
 <div id='side-menu' class="t3_sidebar">
     <ul class="left-menu"><strong>Navigation</strong>
-        <li><a href="{{ site.baseurl }}/static/side-menu.html">Menu contents</a></li>
```

Comment:
It should be valid. This should be the location the menu is pulled from. The big reason I have it there is for web crawlers. We want the content to be discovered.